modules/vector-index/pages/hyperscale-vector-index.adoc

```diff
@@ -0,0 +1,280 @@
+:stem: 
+:page-toclevels: 3
+:index-name: Hyperscale Vector index
+:index-name-plural: Hyperscale Vector indexes
+
+[abstract]
+{description}
+They can scale up to a billion documents containing vectors with a large number of dimensions.
+
+Because they provide the best performance, consider testing a Hyperscale Vector index for your application before resorting to the other types of indexes.
+If you find theirs performance does not meet your needs, then test using a Composite Vector Index or a FTS Vector Index.
+
+
+== How the Hyperscale Vector Index Works 
+
+The Hyperscale Vector Index primarily relies on data stored in an optimized format on disk. 
+By relying on disk storage, they have lower memory requirements than other index types. 
+
+You add a single vector column to a Hyperscale Vector Index. 
+The vector value can be an array of floating point values.
+You can nest the array in another array as long as you use an xref:n1ql:n1ql-language-reference/unnest.adoc[`UNNEST` clause] to extract the it from the containing array. 
+You can also store the vector in a BASE64 string. 
+
+== Prerequisites
+
+Hyperscale Vector Indexes have the following requirements:
+
+* You must have the Index Service enabled on at least one node in your cluster. 
+For more information about how to deploy a new node and Services on your database, see xref:manage:manage-nodes/node-management-overview.adoc[].
+
+* You must have a bucket with scopes and collections in your database. 
+For more information about how to create a bucket, see xref:manage:manage-buckets/create-bucket.adoc[].
+
+* Your account must have the xref:learn:security:roles.adoc#query-manage-index[Query Manage Index] or an administrator role to be able to create an index.
+
+* You have documents in a collection that contain one or more vector embeddings.
+You can add a single vector to a Hyperscale Vector Index.
+If your documents contain multiple embedded vectors, you can create multiple indexes---one for each vector attribute.
++
+Embeddings can be an array of floating point numbers or a base64 encoded string.
+Couchbase {product-name} does not embed vectors itself. 
+You must use an external embedding model to embed vectors into your data and add them to your documents.
++
+[TIP]
+--
+include::vector-search:partial$download-sample-partial.adoc[]
+--
+
+* The vectors you add to an index must contain the same number of dimensions. 
+Also the values in the vector must be 32-bit floating point numbers.
+If a vector does not meet both of these requirements, the vector index treats it as a NULL value and the document is not added to the index. 
+
+* You must know the number of dimensions the vector contains. 
+The embedding model you use to embed the vectors may determine this value for you.  
+For example,  OpenAI API's `text-embedding-ada-002` embedding model creates vectors that have 1536 dimensions.
+
+* You must decide what distance metric and quantization you want your index to use. 
```

Comment:
Distance metric and quantization have defaults - L2_SQUARED and SQ8 

https://docs.google.com/document/d/1c499ChQD8CEU3Lh6UeMHHstgxIgJZGSV6tyGQRjnL9Q/edit?tab=t.0#heading=h.wlm05svaxxtk